src/generated/resources/data/gtceu/tags/blocks/cleanroom_floors.json

```diff
@@ -0,0 +1,6 @@
+{
+  "values": [
+    "gtceu:plascrete",
+    "gtceu:cleanroom_glass"
```

Comment:
I was testing this on 1.20.1-1.5.5 SNAPSHOT 20241206 and I'm not sure that these blocks should be here.

After adding a couple of more block to the tag, I tried placing two of them in the foor, one in the center, and one beside it. With only the center block replaced, the structure forms, but if I replace one of the surronding blocks it doesn't. However, if I remove these blocks from the tag, I can replace the surronding blocks and still have the structure form.

I suspect what is going on is that the blocks above are eating the global max, before falling back to the other alternatives.